app/assets/javascripts/admin_users.js

```diff
@@ -43,10 +43,15 @@
 
   // Hide password fields when non-internal authentication source is selected
   function on_auth_source_change() {
+    var password_fields = jQuery('#password_fields');
+        password_inputs = password_fields.find('#user_password, #user_password_confirmation');
```

Comment:
Identifier 'password_inputs' is not in camel case.<br>Identifier 'password_fields' is not in camel case.<br>'password_inputs' is not defined.